scripts/build.sh

```diff
@@ -0,0 +1,12 @@
+openapi-generator-cli generate \
+		-i ./spec/zlp.yaml \
+		-g typescript-node \
+		-t ./templates/typescript \
+		-o build \
+		--global-property models,supportingFiles \
+		--additional-properties=serviceName=Tokenization \
+		--additional-properties=modelPropertyNaming=original \
+		--additional-properties=helperFunctions=./templates/helpers/camelCase.ts
+
+cp build/model/* src/zalopay/typings
```

Comment:
this `typings` package name is a bit weird to me, can we change to `models`

Reply:
@qtdan just updated